tests/testBasisLists.py

```diff
@@ -182,42 +184,42 @@ def testCentralRegularization(self):
         try:
             self.policyDF.set("centralRegularizationStencil", 1)
             ipDiffim.makeRegularizationMatrix(self.policyDF)
-        except Exception, e:  # stencil of 1 not allowed
+        except Exception as e:  # stencil of 1 not allowed
```

Comment:
OK, I prefer to copy your comments on the tests over to a new ticket. I'll make sure to capture all of them. Thanks!